pkg/cloud/vsphere/actuators/cluster/actuator.go

```diff
@@ -45,7 +45,7 @@ import (
 
 //+kubebuilder:rbac:groups=vsphere.cluster.k8s.io,resources=vsphereclusterproviderspecs;vsphereclusterproviderstatuses,verbs=get;list;watch;create;update;patch;delete
 //+kubebuilder:rbac:groups=cluster.k8s.io,resources=clusters;clusters/status,verbs=get;list;watch;create;update;patch;delete
-//+kubebuilder:rbac:groups=,resources=secrets,verbs=create;get;list;watch
+//+kubebuilder:rbac:groups=,resources=secrets,verbs=create;get;list;watch;delete
```

Comment:
Hi @vijaykatam,

Great catch, thank you! Please make sure you run `make` ahead of submitting the PR. This will regenerate the RBAC files that need to be part of the commit. Additionally, the bootstrap spec will need to be updated as well. To update the bootstrap spec, run `make` and then do the following:

```shell
hack/generate-yaml.sh -f -d -c ci -m '$VSPHERE_CONTROLLER_VERSION' && \
cp -f out/ci/provider-components.yaml scripts/e2e/bootstrap_job/spec/provider-components.template
```

Then, with your editor of choice, please modify `scripts/e2e/bootstrap_job/spec/provider-components.template` by adding `-v=6` as an argument for the CAPV manager's pod spec. For example:

```shell
$ diff out/ci/provider-components.yaml scripts/e2e/bootstrap_job/spec/provider-components.template 
1183a1184
>         - -v=6
```